doc/source/ndl.rst

```diff
@@ -1,8 +1,30 @@
+Naive Discriminative Learning
+=============================
 
-.. _comparison_of_algorithms:
+Terminology
+-----------
+
+Before explaining Naive Discriminative Learning (NDL) in detail, we want to
+give you a brief overview over important notions:
+
+cue :
+    A cue is something that gives a hint on something else. The something else
+    is called outcome. Examples for cues in a text corpus are trigraphs or
+    preceding words for the word or meaning of the word.
+
+outcome :
+    The outcome is the result of an event. Examples are words, the meaning of
```

Comment:
result (stimulus) ?

RW is not a generall Machine Learning architecture, but model of classical conditioning. Therefore we can and  (should in my opinion) stick to the domain specific terms. You might disagree.

Reply:
I opened an issue for that (#133) to resolve it later